### PR TITLE
Fix lost array with container: set

### LINKED ISSUE #64: Framing and @container:@set for scoped contexts
Playground:  http://tinyurl.com/y4ry8o3m

With a very simple identity frame (e.g. give the top node's id and a context), the expected output would be to follow the context's definition for `@container: @set` and wrap single objects in an array.  This works as expected for the context when applied via compaction (see the compaction tab in the playground).

However, in both the javascript and python implementations, the `part` property does not generate an array from the frame, where it does from compaction. Note that it is not due to Production within Production, as changing the class of either to (e.g.) `Activity` results in the same structure.

This seems like a bug?

### PR DESCRIPTION
**Pull request by gkellogg**
* Separate `@preserve` from `@null` processing.
* Add tests for `@null` as a default value.

Fixes #64

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-framing/pull/65.html" title="Last updated on Aug 18, 2019, 4:04 PM UTC (72ee9d2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-framing/65/0ecbc60...72ee9d2.html" title="Last updated on Aug 18, 2019, 4:04 PM UTC (72ee9d2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1668,22 +1668,23 @@ <h3>Framing Algorithm</h3>
   remove the <code>@id</code> <a>entry</a> of each <a>node object</a> where the
   <a>entry</a> value is a <a>blank node identifier</a> which appears only once
   in any property value within <var>result</var>.</p>
-<p>Using <var>result</var> from the recursive algorithm, set <var>compacted results</var> to the result of using the
+<p>Recursively, replace all <a>entries</a> in <var>results</var>
+  where the key is `@preserve` with the first value of that <a>entry</a>.
+  <span class="note">The value of the entry will be an array with a single value,
```
Comment (dlongley) [inline index.html]:
```suggestion
  <span class="note">The value of the entry will be an array with a single value;
```

**Comment by dlongley** [comment]
Thanks, @gkellogg!
